invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/search/SearchResultItemLayout.js

```diff
@@ -51,27 +51,29 @@ class SearchResultItemComponent extends Component {
           data-label="Title"
           className="word-break-all"
         >
-          <BoolFormatter
-            value={result.access.record === "public"}
-            color="green"
-            icon="lock unlocked"
-          />
-          <BoolFormatter
-            value={
-              result.access.record === "restricted" ||
-              result.access.files === "restricted"
-            }
-            color="red"
-            icon="lock"
-          />
-          <a target="_blank" rel="noreferrer" href={result.links.self_html}>
-            {_truncate(result.metadata.title || i18next.t("Empty draft title"), {
-              length: 50,
-            })}
-          </a>
-          <br />
-          <div className="text-muted">
-            {result.id} | {i18next.t("version")} {result.versions.index}
+          <div>
```

Comment:
Why the extra div needed?

Reply:
To fix the mobile view

Reply:
do we understand why adding div is fixing it? it seems like a "hack" not an actua fix

Reply:
So the reason for this behaviour is that to display the column label in every row we override the normal SemanticUI behaviour 

https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides#L111-L112

The highlighted code creates a two column grid where each child is put into alternating columns, and to add the column label as the first child, so the behaviour we are seeing is expected. I think it is reasonable to expect that if you only want one item on the right then you need to pass one child

Reply:
I believe a long term solution should be found, rather than adding divs every time we have several elements in the table cell

Reply:
I don't think it's ideal, but at the moment you could do something like this if you wanted (imagining that an instance supports subtitles)

| title<br>subtitle | Survey of Web Development<br>Focusing on InvenioRDM |
| ---------------- | --------------- |
| author           | Carlin MacKenzie          |

We could "solve" the problem by doing `grid-template-rows:` or changing it to `inline-flex` but in my view it is a bug that the parent element is missing here

Reply:
I think an easier solution would be to set
on the `<td>`
```
display: grid 
``` 

and on the pseudo element before:
```
grid-column:1
```

all the other children of td (by `> *` selector):

```
grid-column:2
```

since in the mobile view we will always have 2 columns

no unexpected behaviours should happen in this way (I think it is a bit similar to what you proposed above)

Reply:
good call! fixed, tested and updated the screenshots above